### DISCUSSION TITLE
export KONG_DATABASE=postgres; docker-compose up fails to start

### DISCUSSION THREAD

**Discussion by amitabak-MA** [Kong/kong#8426] [Help] [answered]
Running the compose variant with postgres  using:    export KONG_DATABASE=postgres; docker-compose up

Fails on 
2022/02/17 14:54:15 [error] 1#0: init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:503: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: host or service not provided, or not known
kong_1                | stack traceback:

Any idea how one makes this work with postgres ?

**Comment by kikito** [comment, marked as answer]
You probably need to set more environment variables, like KONG_PG_HOST, KONG_PG_PORT, and KONG_PG_PASSWORD.
